HISTORY.rst

```diff
@@ -2,6 +2,51 @@
 History
 =======
 
+3.17.2 (2020-09-03)
```

Comment:
If it has entries in the "Changes" section, it probably warrants incrementing the minor version rather than patch.